docs/documentation/case.md

```diff
@@ -677,7 +677,7 @@ The parameters are optionally used to define initial velocity profiles and pertu
 
 - `vel_profile` activates setting the mean streamwise velocity to hyperbolic tangent profile. This option works only for 2D and 3D cases.
 
-- `instability_wave` activates the perturbation of initial velocity by instability waves obtained from linear stability analysis for a mixing layer with hyperbolic tangent mean streamwise velocity profile. This option only works for `n > 0`, `bc_y%[beg,end] = -5`, and ``vel_profile = 'T'``.
+- `instability_wave` activates the perturbation of initial velocity by instability waves obtained from linear stability analysis for a mixing layer with hyperbolic tangent mean streamwise velocity profile. This option only works for `n > 0`, `bc_y%[beg,end] = -6`, and ``vel_profile = 'T'``.
```

Comment:
so this is only for mixing layers? if true, can you be more specific in the documentation here about what a mixing layer is, specifically? you may also want to change `instablility_wave` to something more descriptive, like `mixlayer_instability_trigger`